components/core/src/clp/ffi/ir_stream/IrErrorCode.hpp

```diff
@@ -16,9 +16,10 @@ enum class IrErrorCodeEnum : uint8_t {
     IncompleteStream,
 };
 
-using IrErrorCode = clp::error_handling::ErrorCode<IrErrorCodeEnum>;
+using IrErrorCode = ystdlib::error_handling::ErrorCode<IrErrorCodeEnum>;
+using IrErrorCategory = ystdlib::error_handling::ErrorCategory<IrErrorCodeEnum>;
```

Comment:
```suggestion
```

Move this definition back to `components/core/src/clp/ffi/ir_stream/IrErrorCode.cpp` as it only necessary to the implementation. (I didn't make a suggestion for the line in IrErrorCode.cpp because it is over 100 characters and needs to be formatted, sorry.)